lib/authentication/service/auth_provider.dart

```diff
@@ -58,7 +58,7 @@ class AuthProvider with ChangeNotifier {
 
   void _errorHandler(dynamic e, {bool showToast = true}) {
     try {
-      print(e.message);
+      print('$e.message e.code');
```

Comment:
If you want to print the string message you need to put in `{}`. Example: '${e.message}'. Also, this print will just display "e.code" at the end, not the code. Take a look on this article: https://medium.com/run-dart/dart-dartlang-introduction-string-interpolation-8ed99174119a